lib/models/search.js

```diff
@@ -3,8 +3,9 @@ const fetch = require('node-fetch');
 async function searchBrickset(search) {
   const url = `https://brickset.com/api/v3.asmx/getSets?apiKey=${process.env.API_KEY_BRICKSET}&userHash=&params={'query':'${search}','extendedData':'1',"orderBy":"PiecesDESC"}`;
   const response = await fetch(url);
-  const data = await response.json();
-  return data;
+  const rowData = await response.json();
+  const description = rowData.sets[0].extendedData.description;
```

Comment:
What happens when no bricksets are found?
I guess `rowData.sets[0].extendedData` will throw an error ("Access extendedData of undefined").